### ISSUE TITLE
Expose whole token to usertask

### ISSUE THREAD

**Issue by LeoTT** [process-engine/process_engine#101]
#### Describe your issue

Right now user tasks can only access part of the token.
This must have been a conscious decision at some point, but I fail to see the reason.
As a user I expect access to the same token as in any other task. 

Here is the responsible code:
https://github.com/process-engine/process_engine/blob/bf6f80d6017b3c0809e277c7b0012b787611a2a2/src/entity_types/user_task.ts#L71

#### How can others reproduce the issue?

Try to access token.history in a usertask.
For example:
- uiName: Form
- uiConfig: ${ "message": "why is this undefined " + token.history, "layout": [ { "key": "TOKEN: ", "label": "OK" } ] };

#### Possible solution

Do not cut away part of the token.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by 5ebastianMeier** [comment]
I don't see a good reason why we wouldn't allow history access here.

It doesn't seem to break anything and would only apply to the old implementation anyway.

**Comment by cmg-dev** [comment]
I don't think it is a smart move to expose the whole history to a usertask...

We shall discuss this a little bit.

**Comment by ElRaptorus** [comment]
I did a little digging and found this https://github.com/process-engine/process_engine/issues/47
This really was implemented due to a bug.

But I'm pretty sure, this will no longer be an issue, once we have completely switched to the new object model.

@cmg-dev @5ebastianMeier
